libexec/scoop-analytics.ps1

```diff
@@ -0,0 +1,128 @@
+# Usage: scoop analytics
+# Summary: Collects and sends Scoop usage analytics to a remote server
+# Help: This is an internal command. It is used to collect and send usage analytics
+# to a remote server, at an interval of 7 days. The following data is collected:
+# - Randomly generated (one-time) anonymous ID
+# - Machine info
+#   - OS build number
+#   - OS Architecture
+#   - PowerShell Desktop version
+#   - PowerShell Core version
+#   - Scoop version
+# - Apps installed from public buckets (private apps are filtered out)
+#   - Name
+#   - Version
+#   - Last updated
+#   - Source
+#   - Architecture
+#   - User or Global installation
+#   - Installation status
+# - Public buckets (private buckets are filtered out)
+#   - Name
+#   - Source
+#   - Last updated
+#   - Manifest count
+
+. "$PSScriptRoot\..\lib\json.ps1" # 'ConvertToPrettyJson'
+
```

Comment:
```suggestion
if(!(Resolve-DnsName analytics.scoop.sh -ErrorAction SilentlyContinue)) {
    Write-Host "Could not resolve analytics.scoop.sh"
    exit 0
}

```

Just skip the whole script if analytics.scoop.sh can't be resolved (because it will get added to public blocklists :smile:)

Reply:
Hmm, good idea. But I notice that it takes 10 seconds to time out, isn't that a bit much?

Reply:
Hmm, it returns instantly for me. Maybe because I use a local Adguard in my network. 🤔

Reply:
I noticed that it takes time when connected to my Azure VPN server. On other networks it returns instantly. 

It's useful regardless, so I'll add it.